drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -487,7 +491,7 @@ function DragAndDropBlock(runtime, element, configuration) {
 
             // Set up event handlers:
 
-            $(document).on('keydown mousedown touchstart', closePopup);
+            $element.on('click', "div.item-feedback-popup button.close", closePopup);
```

Comment:
Nit: It's best just to use the class names (instead of including the html elements). You might wish to be consistent with the naming convention of the other button classes (lines 495-497).